ace-link-dashboard.el

```diff
@@ -75,7 +77,10 @@
       (goto-char (window-start))
       (while (< previous-point (funcall next-widget-point))
         (setq previous-point (point))
-        (push (cons (widget-at previous-point) previous-point) candidates))
+        (push (cons (widget-at previous-point) (if (eq 'unicode (char-charset (char-after (point))))
```

Comment:
Could you put the `if` in a new line?
Or better, move that conditional to a new function